src/Widgets/FormInput/FormInput.tsx

```diff
@@ -239,8 +238,8 @@ export const FormInput = forwardRef<FormSubmit, FormInputProps>((props, ref):JSX
                         validate: (value, formValues) => {
                             if (fieldValueIsString(field.type, value)) {
                                 if (field.field == 'dateOfBirth') {
-                                    const minAge:Date = getDOBMaxDate(userRole || RoleEnum.STUDENT);
```

Comment:
When I change the above to RoleEnum[userRole] || RoleEnum.STUDENT typescript doesn't like it.

Element implicitly has 'any' type because expression of type 'string' cannot be used to index type 'typeof RoleEnum'

Reply:
Try this:

`RoleEnum[userRole as keyof typeof RoleEnum] || RoleEnum.USER;`